lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -0,0 +1,121 @@
+from abc import ABC, abstractmethod
+
+from sycamore.plan_nodes import Node
+
+from sycamore.data import Document
+from sycamore.transforms.map import Map
+from dateutil import parser
+
+
+class Standardizer(ABC):
+
+    @abstractmethod
+    def standardize(self, document: Document) -> Document:
+        pass
+
+
+class LocationStandardizer(Standardizer):
+    """
+    This Class standardizes the format of US state location.
+    """
+
+    def __init__(self):
```

Comment:
This is fine for now but I wonder if an existing package like `address` could be used for this kind of thing.

https://pypi.org/project/address/